src/paradigma/config.py

```diff
@@ -375,9 +373,8 @@ def __init__(self, min_window_length: float = 10) -> None:
         super().__init__()
 
          # Parameters for HR analysis
-        self.window_length_s: int = 6
-        self.window_step_size_s: int = 1
-        self.min_hr_samples = min_window_length * self.sampling_frequency
+        self.window_overlap_s = self.window_length_s - self.window_step_size_s
+        self.min_hr_samples = int(min_window_length * self.sampling_frequency)
```

Comment:
E.g., if sampling frequency equals 25 Hz, and window length equals 3.5, then we have  `min_window_length * self.sampling_frequency` equals 25 * 3.5 = 87.5. Now, inconveniently `int(87.5)` equals 87 (`int` rounds to the lowest integer). So perhaps, for these edge cases, you want to use `int(np.round(min_window_length * self.sampling_frequency))` or something along these lines.

Reply:
Goed punt! En zal het aanpassen, heb je bezwaren tegen round i.p.v. np.round omdat je anders in de config numpy moet inladen als module @Erikpostt

Reply:
There are some [minor differences](https://stackoverflow.com/questions/20388071/what-are-the-under-the-hood-differences-between-round-and-numpy-round) between the two methods, and I don't think that having to import `numpy` should restrict us to using `round()`. I'll let you decide on this matter.

Reply:
ALlright, but it doesn't seem that np.round is preferred over round() right? So than we can use just round() @Erikpostt

Reply:
No they seem to have different methods for rounding, but I expect that, in case the rounding is ever done differently, it should not affect the results much.